Makefile

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BINARY_PATH         := bin/
+COVERPROFILE        := test/output/coverprofile.out
+IMAGE_REPOSITORY    := <link-to-image-repo>
+IMAGE_TAG           := $(shell cat VERSION)
+PROVIDER_NAME       := SampleProvider
+PROJECT_NAME        := gardener
+CONTROL_NAMESPACE  := default
+CONTROL_KUBECONFIG := dev/target-kubeconfig.yaml
+TARGET_KUBECONFIG  := dev/target-kubeconfig.yaml
+
+#########################################
+# Rules for running helper scripts
+#########################################
+
+.PHONY: rename-provider
+rename-provider:
+	@./hack/rename-provider ${PROVIDER_NAME}
+
+.PHONY: rename-project
+rename-project:
+	@./hack/rename-project ${PROJECT_NAME}
+
+#########################################
+# Rules for starting machine-controller locally
+#########################################
+
+.PHONY: start
+start:
+	@GO111MODULE=on go run \
+			-mod=vendor \
```

Comment:
This is actually generated from the script from the SampleProvider. Since most of these things are actually generated in the Makefile I left it as it is.

Reply:
I think it's redundant, so please remove it.